spec/models/user_spec.rb

```diff
@@ -31,7 +41,8 @@
   describe '#email' do
     it 'must be unique' do
       create(:user, email: 'user@example.org')
-      expect { build(:user, email: 'user@example.org').save!(validate: false) }.to raise_error(ActiveRecord::RecordNotUnique)
```

Comment:
@roschaefer What was the reason you chose to test for the database exception, not the validation exception?

Reply:
To ensure the unique constraint is in the database